src/ca/mcgill/cs/jetuml/views/StringViewer.java

```diff
@@ -34,6 +42,8 @@
  * - bold
  * - different alignments.
  */
+@Immutable 
+@Flyweight

```

Comment:
Is it fine to have these annotations? I recall at the beginning of the semester you mentioned you created them after having refactored the `GeomUtil` classes

Reply:
Yes, that was the idea.